server/src/main/kotlin/com/kamelia/jellyfish/util/FileUtils.kt

```diff
@@ -0,0 +1,55 @@
+package com.kamelia.jellyfish.util
+
+import com.kamelia.jellyfish.core.UploadCodeGenerationException
+import com.kamelia.jellyfish.rest.file.Files
+import io.ktor.http.content.PartData
+import io.ktor.http.content.streamProvider
+import java.io.File
+import java.nio.file.Path
+import java.util.UUID
+import kotlinx.coroutines.Dispatchers
+import kotlinx.coroutines.withContext
+
+object FileUtils {
+
+    private val UPLOAD_PATH = Path.of("upload")
+
+    init {
+        java.nio.file.Files.createDirectories(UPLOAD_PATH)
+    }
+
+    fun getOrNull(owner: UUID, code: String): File? {
+        val file = UPLOAD_PATH.resolve(owner.toString()).resolve(code).toFile()
+        return if (file.exists()) file else null
+    }
+
+    suspend fun write(
+        owner: UUID,
+        file: PartData.FileItem
+    ): Triple<String, String, Long> = withContext(Dispatchers.IO) {
```

Comment:
This method absolutely needs documentation. Generally, we need to write documentation along with new methods so that anybody can understand quickly what's the point of every method.